docs/source/quick start.rst

```diff
@@ -82,66 +82,66 @@ Warning: The first of the following commands is not well tested. If you run into
+Before starting, you may want to make sure everything functions as intended. I suggest you to try running some of the examples in the `official GitHub repository of the project <https://github.com/CoffeeStraw/PyonFX/tree/master/examples>`_.
 
-To run a script in python, all you need to do is run the following command:
+To run a script in python, run the following command:
 
 .. code-block:: sh
    :emphasize-lines: 1
 
    python namefile.py
 
-Or if this is not working for some reason (like you're not on Windows and both Python2 and Python3 are installed):
+Or if this for some reason doesn't work (like if you're not on Windows and both Python2 and Python3 are installed):
 
 .. code-block:: sh
    :emphasize-lines: 1
 
    python3 namefile.py
 
-I highly suggest you to generate and study every single example in this examples folder (download always up-to-date `here <https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/CoffeeStraw/PyonFX/tree/master/examples>`_). These are meant for absolute beginners until advanced users and explain in detail the usage of all the relevant functions of the library.
+I highly suggest you generate and study every single example in the examples folder (download always up-to-date `here <https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/CoffeeStraw/PyonFX/tree/master/examples>`_). These are meant to help out beginners to advanced users by explaining all the relevant functions of the library and how they work in detail.
 
 Tips
 ++++
 
-* Don't make a KFX in one go. Make pauses, go for a walk, collect ideas from your surroundings;
-* Pick elements of the video. Your effect should merge with the background in some manner;
-* Consider human recognition. Mostly we notice motion, then contrasts, then colors. Too much can give a headache, too less is boring;
+* Don't make KFX in one go. Take breaks, go for a walk, obtain inspiration from your surroundings;
+* Pick elements of the video. Your effects should ideally blend in with the video;
+* Consider human recognition. Humans notice motion first, then contrasts, then colors. Too much of any of this can result in headaches, but too little can be boring to look at;
 * Use modern styles to impress (light, curves, particles, gradients) and old ones for readability (solid colors, thick borders, static positions);
-* When background is too flashy, try to insert a panel shape to put your text on 'safe terrain';
-* Adjust to karaoke times and voice. Fast sung lines haven't syllable durations for effects which need some time to get seen.
+* When backgrounds are too flashy, try to insert a panel shape to put your text on 'safe terrain';
+* Adjust to karaoke timing and voice. Fast sung lines will have very short syllable durations for effects, and may not always be visible.
 
 ----------
 
 .. rubric:: Footnotes
 .. [#f1] List of all ASS tags with usage explanation: http://docs.aegisub.org/3.2/ASS_Tags/
```

Comment:
This site is currently down. A partial mirror can be found [here](https://aegisub.shinon71.moe/), but it might also be worth waiting for the official site to come back online again before updating it.

Reply:
Should we put both for now?

Reply:
That's an option, but I'm not sure if the new docs will be hosted on the same address.

Reply:
Could linking this cached version be a good solution? https://web.archive.org/web/20200722050630/http://docs.aegisub.org/3.2/ASS_Tags/

Reply:
Yup, that sounds like a good idea. Maybe stick with that for now and update it to the new one when that goes up instead of Shinon's mirror?

Reply:
Yeah, I agree!